py/src/braintrust/framework.py

```diff
@@ -179,9 +179,12 @@ class EvalScorerArgs(SerializableDataClass, Generic[Input, Output]):
 
 OneOrMoreScores = Union[float, int, bool, None, Score, List[Score]]
 
+# Type representing a scorer-like object with eval_async method
+ScorerLike = Any  # Any object with appropriate eval methods
```

Comment:
This perhaps could be a python protocol, but these are just type hints, so I think this is ok.

Reply:
I think if it's part of a public type signature it might be good to make this more specific, because we're essentially weakening the public interface to accept anything as a scorer.